<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>129</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_books">
       <property name="text">
        <string>Gestionar libros</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_users">
       <property name="text">
        <string>Gestionar usuarios</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_loans">
       <property name="text">
        <string>Gestionar pr√©stamos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_laptops">
       <property name="text">
        <string>Gestionar laptops</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
